### PR TITLE
Improve "Shards" view in web UI

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Improve "Shards" view in web UI so that the shards of individual collections can be expanded and collapsed without affecting the display of any other shards. Also added a "Toggle all" button the web UI to expand/collapse the shards for all collections.

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [x] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [x] :book: CHANGELOG entry made

#### Backports:

- [x] No backports required

### Testing & Verification

- [x] This change is a trivial rework / code cleanup without any test coverage.
- [x] The behavior in this PR was *manually tested*

### CODE REVIEW

**Comment by hkernbach** [comment]
@KVS85 fyi: Will review the PR as soon as @jsteemann checked my last additional commit here:
- https://github.com/arangodb/arangodb/pull/14148

As this is also related to this PR then.

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/shardsView.js [hunk 1]
```diff
@@ -45,20 +46,48 @@
...
-    toggleSections: function (e) {
+    toggleSection: function (e) {
       var colName = $(e.currentTarget).parent().attr('id');
-      this.visibleCollection = colName;
-      $('.sectionShardContent').hide();
-      $(e.currentTarget).next().show();
-      this.renderArrows(e);
-
-      this.getShardDetails(colName);
+      var wasVisible = (this.visibleCollections.indexOf(colName) !==-1);
```
Comment (hkernbach) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/shardsView.js:75]:
```suggestion
      var wasVisible = (this.visibleCollections.indexOf(colName) !== -1);
```

**Review by hkernbach — APPROVED** [review]
LGTM, just found one minor thing (whitespace / readability)
The generated files have not been review ;)

**Comment by jsteemann** [comment]
@KVS85 : PR is almost ready now, but still the UI files need to be regenerated once again.

**Pull request merged**